ExodusMovement/suppress-eslint-errors
# feat: merge upstream changes to support eslint 8

## Test Plan

1. clone this repo at this branch
2. check out HEAD~1 on this PR https://github.com/ExodusMovement/exodus-mobile/pull/16953
3. run `../suppress-eslint-errors/bin/index.js ./src/_local_modules --rules 'import/no-unused-modules'`, see that it suppresses offenders
